src/export_off.cc

```diff
@@ -83,7 +83,7 @@ void export_off(const shared_ptr<const Geometry> &geom, std::ostream &output)
 	append_geometry(geom, mesh);
 
 	output << "OFF " << mesh.vertices.size() << " " << mesh.numfaces << " 0\n";
-	const Vector3d *v = mesh.vertices.getArray();
+	const auto v = mesh.vertices.getArray();
```

Comment:
I'm pretty sure that `const auto v` converts to `const std::vector<Vector3d> v`, causing a copy to be made. `const auto &v` would force a reference to be used. Needs testing though as I'm not 100% sure.

Reply:
Argh, really? That's an ugly trap then. I'll try to find out...

Reply:
Right, so according to https://isocpp.org/blog/2018/09/quick-q-cpp11-auto-what-if-it-gets-a-constant-reference the const is not really needed, but the  `&`.

Reply:
Heh, Compiler Explorer with binary execution support is just awesome: https://godbolt.org/z/CyxTL_ :)